kqueen/blueprints/api/test_user.py

```diff
@@ -34,7 +34,7 @@ def get_auth_headers(self):
         Returns:
             dict: Authorization header dict
         """
-        data = {'username': self.obj.username, 'password': self.obj.password}
+        data = {'username': self.obj.username, 'password': self.obj.username + 'password'}
```

Comment:
Why do we need to pass self.obj.username twice?

Reply:
Well, we don't. I removed the fake password, because it's random, there is no simple way to pass it along with the user object and `user.password` no longer returns value you could use for getting auth header. So I changed test users password to his username + the string 'password'. We can find another way, but this was quick fix for getting auth header in tests.

Reply:
Oh, it's just test. Fine, I understand it now.